### PR TITLE
[tests] fixup dbcrash interaction with add_nodes()

### PR DESCRIPTION
**Pull request by jnewbery**
Another conflict with #11121. Apologies - this is entirely my fault. I didn't run the extended test suite after rebasing on master.

@MarcoFalke @sdaftuar

### CODE REVIEW

File: test/functional/dbcrash.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def set_test_params(self):
 
     def setup_network(self):
         # Need a bit of extra time for the nodes to start up for this test
-        self.add_nodes(self.num_nodes, timewait=90)
+        self.add_nodes(self.num_nodes, extra_args=self.extra_args, timewait=90)
```
Comment (MarcoFalke) [inline test/functional/dbcrash.py]: We might as well get rid of passing member variables to methods.

**Review by maflcko — COMMENTED** [review]
utACK 8fdb6f9126fe1e13b60de11fd34c204247679ef4

**Comment by sdaftuar** [comment]
ACK, this fixes `dbcrash.py` now

File: test/functional/dbcrash.py (hunk 1 above)
Comment (jnewbery) [reply test/functional/dbcrash.py]: Agree, but I haven't looked into how big a change that would be (I'd need to look at all calls to `add_nodes()` to make sure they're not calling this method with no `extra_args` method and expecting the node to be started with any `extra_args` equal to `[]`)

**Pull request merged**